### PR TITLE
Switch to Gradle.

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

**Comment by JakeWharton** [comment]
I have another module which I want to open source which is an Android library.

**Comment by JakeWharton** [comment]
There's an implicit "reluctantly" on the end of this PR title

File: checkstyle.xml [hunk 1]
```diff
@@ -1,120 +0,0 @@
-<?xml version="1.0"?>
```
Comment (f2prateek) [inline checkstyle.xml]: Should bring this back with https://github.com/f2prateek/android-checkstyle-plugin

**Comment by f2prateek** [comment]
LGTM

File: checkstyle.xml (hunk 1 above)
Comment (JakeWharton) [inline checkstyle.xml]: Feel free. It was annoying me so I just removed it.

**Pull request merged**